core/src/main/java/org/apache/accumulo/core/data/Mutation.java

```diff
@@ -1237,4 +1238,37 @@ protected SERIALIZED_FORMAT getSerializedFormat() {
     return this.useOldDeserialize ? SERIALIZED_FORMAT.VERSION1 : SERIALIZED_FORMAT.VERSION2;
   }
 
+  /**
+   * Creates a multi-lined, human-readable String for this mutation.
+   *
+   * This method creates many intermediate Strings and should not be used for large volumes of
+   * Mutations.
+   *
+   * @return A multi-lined, human-readable String for this mutation.
+   */
+  public static String prettyPrint(Mutation mutation) {
+    StringBuilder sb = new StringBuilder();
+
+    sb.append("mutation: ").append(utf8String(mutation.row)).append('\n');
+    for (ColumnUpdate update : mutation.getUpdates()) {
+      sb.append(" update: ");
+      sb.append(utf8String(update.getColumnFamily()));
+      sb.append(':');
+      sb.append(utf8String(update.getColumnQualifier()));
+      sb.append(" value ");
+
+      if (update.isDeleted()) {
+        sb.append("[delete]");
+      } else {
+        sb.append(utf8String(update.getValue()));
+      }
+      sb.append('\n');
+    }
+
+    return sb.toString();
+  }
+
+  private static String utf8String(byte[] bytes) {
+    return new String(bytes, StandardCharsets.UTF_8);
```

Comment:
If you just do a static import of `StandardCharsets.UTF_8`, the lines where this is used become short enough that you don't need this separate private helper method.